inspirehep/modules/search/query_suggestions.py

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Yeah it makes sense to split the files. But since we use the walkers from the search module to give suggestions and not from the query-parser I am not sure if the signals should be in query-parser or the module.